vmm/src/vm.rs

```diff
@@ -651,7 +646,6 @@ impl Vm {
         })
     }
 
-    #[cfg(feature = "acpi")]
```

Comment:
@rbradford , is there any problem on X86 if we make NUMA not specific to ACPI?

Reply:
Should be fine, the `--numa` parsing is not feature gated.